source/autocomplete/tests/test.cpp

```diff
@@ -741,9 +741,12 @@ BOOST_AUTO_TEST_CASE(autocomplete_functional_test_admin_SA_and_Address_test) {
     b.data->pt_data->sort_and_index();
 
 
-    b.add_way("rue DU TREGOR", "");
-    b.add_way("rue VIS", "");
-    b.add_way("quai NEUF", "");
+    auto w = b.add_way("rue DU TREGOR", "");
+    w->visible = true;
```

Comment:
Why not overloading `add_way()` in `build_helper.cpp` to set the field `w->visible = true` by default ? That would reduce the changes.

```cpp
navitia::georef::Way* builder::add_way(const std::string& name, const std::string& way_type, bool visible = true){...}
```

Reply:
OK :eagle: